spec/factories/districts.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :district do
+    name 'Back Bay'
+    description 'A wonderful location in the heart of Greater Boston.'
+    user_id 1
+  end
+end
```

Comment:
To make your testing more robust, rather than specifying a `user_id`, your `district` factory should invoke a `user` factory.